src/icons/fileIcons.ts

```diff
@@ -1968,7 +1968,7 @@ export const fileIcons: FileIcons = {
     {
       name: 'nginx',
       fileNames: ['nginx.conf'],
-      fileExtensions: ['nginx', 'nginxconfig'],
+      fileExtensions: ['nginx', 'nginxconf'],
```

Comment:
What about just adding it so doesn’t break existing files with the old name`

Reply:
yes, please just add it and don't remove existing file extensions